lib/pure/math.nim

```diff
@@ -357,3 +391,36 @@ when isMainModule and not defined(JS):
   assert(lgamma(1.0) == 0.0) # ln(1.0) == 0.0
   assert(erf(6.0) > erf(5.0))
   assert(erfc(6.0) < erfc(5.0))
+when isMainModule:
+  # Function for approximate comparison of floats
+  proc floatIsEqual(x, y: float): bool = (abs(x-y) < 1e-9)
```

Comment:
Hmmm... okay.  I don't think I'd seen that one before.  There doesn't seem to be an applicable reference in the index ([http://nim-lang.org/docs/theindex.html#=~](http://nim-lang.org/docs/theindex.html#=~)).  Where can I find out about what `=~` and `==~` mean?